docs/self-managed/modeler/web-modeler/configuration/database.md

```diff
@@ -0,0 +1,89 @@
+---
+id: database
+title: "Database"
+description: "Read details on how to connect Web Modeler with a database."
+---
+
+:::note
+Web Modeler Self-Managed is available to [enterprise customers](../../../../reference/licenses.md#web-modeler) only.
+:::
+
+Web Modeler requires a PostgreSQL database as persistent data storage (other database systems are currently not supported).
+Use the following environment variables of the `restapi` component to set up your database connection:
+
+| Environment variable                  | Description                                           | Example value                                            |
+| ------------------------------------- | ----------------------------------------------------- | -------------------------------------------------------- |
+| `SPRING_DATASOURCE_URL`               | JDBC URL of the database                              | `jdbc:postgresql://postgres.example.com:5432/modeler-db` |
+| `SPRING_DATASOURCE_USERNAME`          | Database user name                                    | `modeler-user`                                           |
+| `SPRING_DATASOURCE_PASSWORD`          | Database user password                                | \*\*\*                                                   |
+| `SPRING_DATASOURCE_DRIVER_CLASS_NAME` | [optional]<br/>Java class name of the database driver | `software.amazon.jdbc.Driver`                            |
```

Comment:
Since `SPRING_DATASOURCE_DRIVER_CLASS_NAME` is optional you could add a new column "Default value" with the value `org.postgresql.Driver`.

Reply:
Technically that's not correct though, if `SPRING_DATASOURCE_DRIVER_CLASS_NAME` is not set, Spring Boot automatically derives the driver from the JDBC URL (via [org.springframework.boot.jdbc.DatabaseDriver](https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/jdbc/DatabaseDriver.java). 

I'd refrain from adding a default value here in order not to entice users to add an explicit value and thus override Spring Boot's autoconfiguration mechanism. Wdyt?

Reply:
OK, good point. Then let's don't add a default value. Most users won't need to care about `SPRING_DATASOURCE_DRIVER_CLASS_NAME` anyway.